### PR TITLE
James 245 display multiple players in page from list

### PR DESCRIPTION
**Pull request by JamesPHoughton**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -54,18 +52,20 @@ export default function App() {
...
 
-  // the second player in this block lets us cypress test multiple players at the
-  // same time. 
-  return (
-    <div className="h-screen relative">
-      {dev && <EmpiricaMenu />}
-      <div className="h-full overflow-auto">
-        <div test-player-id="player1">
+  function renderPlayers (playerKeys) {
+    var players = [];
```
Comment (Alan-Qiao) [inline client/src/App.jsx:65]: const players = [];

File: cypress/support/commands.js [hunk 2]
```diff
@@ -231,28 +235,45 @@ Cypress.Commands.add('empiricaLoginMultiPlayers', (playerKeys) => {
...
+      
+      // Submit
+      cy.wrap($player, { log: false }).find("button", { log: false }).contains("I AGREE", { log: false }).click({ log: false });
+
+      // Login
+      cy.wrap($player, { log: false }).contains("Enter your", { timeout: 5000, log: false, matchCase: false });
+      cy.wrap($player, { log: false }).find("input", { log: false }).click({ log: false }).type(playerKey, { log: false });
+      cy.wrap($player, { log: false }).find("button", { log: false }).contains("Enter", { log: false }).click({ log: false });
+    } )
+  })
```
Comment (Alan-Qiao) [inline cypress/support/commands.js:276]: I like this wrap syntax! It's pretty. Do you have to wrap it everytime or is it possible to define a variable and just run .contains on it?

**Review by Alan-Qiao — APPROVED** [review]
Looks good!

File: cypress/support/commands.js (hunk 2 above)
Comment (JamesPHoughton) [reply cypress/support/commands.js:276]: Good question! I don't know - I'll look it up.

**Pull request merged**